lib/mix/lib/mix/tasks/profile.fprof.ex

```diff
@@ -126,7 +126,9 @@ defmodule Mix.Tasks.Profile.Fprof do
 
     # Start app after rewriting System.argv,
     # but before requiring and evaling
-    Mix.Task.run "app.start", args
+    unless opts[:start] == false do
```

Comment:
`unless "--no-start" in args` is the pattern we use throughout Mix for this one.